client/components/gsuite/gsuite-new-user-list/style.scss

```diff
@@ -0,0 +1,97 @@
+.gsuite-new-user-list__new-user-remove-user-button {
+	width: 100%;
+	@include breakpoint( '>800px' ) {
+		width: 75px;
+	}
+
+	@include breakpoint( '>800px' ) {
+		span {
+			display: none;
+		}
+	}
+}
+
+.gsuite-new-user-list__actions {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+
+	@include breakpoint( '>480px' ) {
+		flex-direction: row;
+	}
+
+	@include breakpoint( '>660px' ) {
+		flex-direction: column;
+	}
+
+	@include breakpoint( '>800px' ) {
+		flex-direction: row;
+	}
+}
+
+.gsuite-new-user-list__add-another-user-button {
+	width: 100%;
+	@include breakpoint( '>480px' ) {
+		width: auto;
+	}
+}
+
+.gsuite-new-user-list__new-user-domain-select {
+	width: 100%;
+	@include breakpoint( '>800px' ) {
+		width: auto;
+	}
+}
+
+.gsuite-new-user-list__user-divider {
+	width: 100%;
+	margin: 1.5em 0;
+}
+
+.gsuite-new-user-list__new-user-name {
+	@include breakpoint( '>800px' ) {
+		display: flex;
+		justify-content: space-between;
+	}
+}
+
+.gsuite-new-user-list__new-user-email {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+
+	// input {
+	// 	margin: 0 0 15px;
+	// 	width: auto;
+	// }
```

Comment:
We probably want to remove those commented lines.

Reply:
fixed